sycl/doc/design/opencl-extensions/cl_intel_global_variable_access.asciidoc

```diff
@@ -292,6 +280,29 @@ Add two new rows to Table 37, *List of supported event command types*:
 |===
 
 
+== Modifications to the OpenCL SPIR-V Environment Specification
+
+=== New Section "Global Variables"
+
+Add a new subsection under section 2, *Common Properties* named *Global
+Variables* with the following content:
+
+Host code may read or write the content of a global variable in a `cl_program`
+by calling *clEnqueueReadGlobalVariableINTEL* or
+*clEnqueueWriteGlobalVariableINTEL*.  Those two functions both take a _name_
+parameter which identifies the variable.  This parameter is interpreted as
+follows:
```

Comment:
I'd consider saying explicitly that this section only applies for programs created via SPIR-V, perhaps by adding to the last sentence: 

> **For a `cl_program` created from SPIR-V,** this parameter is interpreted as follows...

Debatably this is unnecessary because this section is in the SPIR-V environment specification, but I still think it is clearer to say it explicitly.

Reply:
Makes sense.  Done in 39cad4d97270986c2c03c2fd185045019d5fbc75